test/acceptance/features/rabbit.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Rabbit client
+
+  @rabbit
+  Scenario: Publish and subscribe a text message
+    Given the rabbit endpoint "amqp://guest:guest@localhost:5672/"
+      And I subscribe to the rabbit topic "test-topic"
+     When I publish a message with the correlator "test-correlator" to the rabbit topic "test-topic" with the text
+          """
+          This is a test message
+          """
+     Then I wait up to "3" seconds for a rabbit message with the correlator "test-correlator" with the text
+          """
+          This is a test message
+          """
+     And I unsubscribe from the rabbit topic "test-topic"
```

Comment:
Instead of explicitly unsubscribing to the topic, wont it be better to do implicitly using [Suite#AfterScenario][1] hook?

[1]: https://pkg.go.dev/github.com/data-dog/godog#Suite.AfterScenario